pipelines/corefxlab-base.yml

```diff
@@ -3,25 +3,58 @@ parameters:
   displayName: ''
   targetOS: ''
   pool: {}
+  isOfficialBuild: false
 
 jobs:
   - job: ${{ parameters.name }}
     displayName: ${{ parameters.displayName }}
     variables:
-      ${{ if eq(parameters.targetOS, 'Windows_NT') }}:
-        _buildScript: build.cmd
-      ${{ if ne(parameters.targetOS, 'Windows_NT') }}:
-        _buildScript: ./build.sh
+      - ${{ if eq(parameters.targetOS, 'Windows_NT') }}:
+        - name: _buildScript
+          value: build.cmd
+      - ${{ if ne(parameters.targetOS, 'Windows_NT') }}:
+        - name: _buildScript
+          value: ./build.sh
+      
+      - ${{ if eq(parameters.isOfficialBuild, 'true') }}:
+        - name: _buildArguments
+          value: -Configuration $(_BuildConfig) -BuildVersion $(Build.BuildNumber)
+        - name: _SignType
+          value: real
+        - name: _TeamName
+          value: DotNetCore
+        - group: DotNet-MyGet-Publish
+
+      - ${{ if eq(parameters.isOfficialBuild, 'false') }}:
+        - name: _buildArguments
+          value: $(_BuildConfig)
 
     pool: ${{ parameters.pool }}
 
     strategy:
       matrix:
-        x64_Debug:
-          _BuildConfig: Debug
+        ${{ if eq(parameters.isOfficialBuild, 'false') }}:
+          x64_Debug:
+            _BuildConfig: Debug
+
         x64_Release:
           _BuildConfig: Release
 
     steps:
-      - script: $(_buildScript) $(_BuildConfig)
+      - ${{ if eq(parameters.isOfficialBuild, 'true') }}:
+        - task: MicroBuildSigningPlugin@2
+          displayName: Install MicroBuild plugin
+          inputs:
+            signType: $(_SignType)
+            zipSources: false
```

Comment:
**Question:** What does this do?

Reply:
Do you mean the zipSources parameter? I have no idea, I just copied what arcade does.